src/v1/table/generics/primaryKey.ts

```diff
@@ -6,21 +6,23 @@ import { HasSK } from './hasSk'
 /**
  * Returns the TS type of a Table Primary Key
  *
- * @param T Table
+ * @param TableInput Table
  * @return Object
  */
-export type PrimaryKey<T extends TableV2 = TableV2> = TableV2 extends T
+export type PrimaryKey<TableInput extends TableV2 = TableV2> = TableV2 extends TableInput
```

Comment:
imo the word input is redundant here

Reply:
It won't be when `TableV2` will become `Table` ^^